### ISSUE TITLE
When using "Hanging Verse Numbers (in Poetry)", 2-digit numbers should be right-aligned (Linux)

### ISSUE THREAD

**Issue by KimB2017** [sillsdev/ptx2pdf#210]
Using today's repo version of ptxprint in Wasta 18.04. 
I like the hanging verse number option for poetry. However, 2-digit verse numbers should be right-aligned; currently they cause the first word of the verse to be indented a bit more than in single-digit verses. I can send a screenshot if you need it.

**Comment by davidg-sil** [comment]
In my own sample file, and with some more debugging code inserted, I'm seeing that hanging of verse numbers is not being applied consistently. E.g. Ps 1 has verse numbers hanging, but Ps 2, on the same page does not. 

To test this, if you have any poetry lines that are \q1 \v1 textline1  .... \q1 textline2, then if hanging is working correctly, then 'textline1' and 'textline2' will start at the same point (irrespective of the presence of the verse number).

Can you check if, in the places where 2 digit numbers are incorrect, the verse numbers are actually hanging at all? (this may need you to temporarily modify the USFM, sorry).

**Comment by KimB2017** [comment]
Oops! I'm sorry—in checking my USFM I realized some marker changes I thought I'd made were not saved. When I made sure my q markers were correct the hanging verse numbers work perfectly for me throughout MAT 1-6. See my attached sample PDF output.

One thing I've noticed is that the marker \qm does not seem to be recognized as "poetry". I tried to use \qm in the genealogy list in MAT 1, but when I added formatting in PrintDraft-mods.sty similar to my q1, q2 formats, the left margin stuck out way to the left of the page margin! Maybe this isn't the intended use of qm; I haven't checked the USFM master doc lately. In this sample PDF I'm using q2 for the genealogy list, and q1 + q2 in the quoted bits in ch 5-6.
[ptxprint-MATRTTT.pdf](https://github.com/sillsdev/ptx2pdf/files/5246302/ptxprint-MATRTTT.pdf)
